public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -474,6 +474,13 @@ export class CloudWatchDatasource
     return logGroupNames;
   }
 
+  async describeAllLogGroups(params: DescribeLogGroupsRequest): Promise<string[]> {
+    const dataFrames = await lastValueFrom(this.makeLogActionRequest('DescribeAllLogGroups', [params]));
+
+    const logGroupNames = dataFrames[0]?.fields[0]?.values.toArray() ?? [];
```

Comment:
Unrelated to this PR of course, but at some point we should move the DescribeLogsGroups request to the resource handler so we don't have to do this mad parsing from data frames.